docs/platform/concepts/service-integration.md

```diff
@@ -2,15 +2,20 @@
 title: Service integrations
 ---
 
-Service integrations provide additional functionality and features by
-connecting different Aiven services together. This includes metrics
-integration which enables Aiven users to send advanced telemetry data to
-an [Aiven for PostgreSQL®](https://aiven.io/postgresql) database for
-metrics and visualize it in [Aiven for
-Grafana®](https://aiven.io/grafana). In addition to the metrics
-integration, log integration is supported that allows sending logs from
-any Aiven service to [Aiven for
-OpenSearch®](https://aiven.io/opensearch).
+Service integrations provide additional functionality and features by connecting different Aiven services together.
+
+## About service integrations
+
+Service integrations include metrics integration which enables Aiven users to send
+advanced telemetry data to an [Aiven for PostgreSQL®](https://aiven.io/postgresql)
+database for metrics and visualize it in [Aiven for Grafana®](https://aiven.io/grafana).
+In addition to the metrics integration, log integration is supported that allows sending
+logs from any Aiven service to [Aiven for OpenSearch®](https://aiven.io/opensearch).
```

Comment:
Would it make sense to also mention as an example the use of PG foreign data wrapper? Just to familiarize the audience with the IP Allow-list integrations.

Reply:
Sounds like a good idea indeed, but in this PR we're addressing specific issues as described in the ticket so maybe we should have a separate ticket for this one? Or do you prefer to add here?
In any case, please provide me with the details on the use of PG foreign data wrapper you'd like to add.

Reply:
I am afraid I don't have the best understanding of such details, but the more important point was to mention that a Service Integration should be used when connecting a service to another that's not publicly accessible :+1: